source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -83,21 +84,74 @@ GLSSharpNavierStokesSolver<dim>::generate_cut_cells_map()
     {
       if (cell->is_locally_owned() || cell->is_ghost())
         {
-          bool cell_is_cut;
+          bool         cell_is_cut;
+          bool         cell_is_inside;
+          unsigned int p_id_cut    = 0;
+          unsigned int p_id_inside = 0;
           // is the particle index that cut the cell if it's cut.  If the cell
           // is not cut the default value is stored (0). If the cell is not cut
           // this value will never be used.
-          unsigned int p;
 
-          std::tie(cell_is_cut, p, local_dof_indices) =
-            cell_cut(cell, local_dof_indices, support_points);
+          // Check if a cell is cut and if it's rerun the particle by which it's
+          // cut and the local DOFs index. The check is done by counting the
+          // number of DOFs that is on either side of the boundary define by a
+          // particle.
```

Comment:
This comment does not make sense. Does it describe what is being done after or the goal of the whole function? Please clean it.